ci/test-locally.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -ex
 
-python3 -m pip --user install -U ../batch
+python3 -m pip install --user -U ../batch
```

Comment:
Does --user need to be in the Dockerfile as well?

Reply:
nah that happens in a docker image with root privileges. our tests don't run as root.